wiki/History_of_osu!/Mapping_and_Modding_Timeline/en.md

```diff
@@ -526,3 +526,75 @@ Mapping and modding systems are constantly improving. The **Mapping and modding
+- **2021-12-14:** The burai sliders rule has been loosened per this [proposal](https://osu.ppy.sh/community/forums/topics/1442020).
+- This was done to accomodate for new mapping styles by making specific criteria for each difficuly, instead of making it a general rule.
+
+**4500 beatmaps** were ranked in 2021.
+
+## 2022
+
+### January
+
+- **2022-01-21:** A new feature dedicated to making BN requests more convenient has been deployed to [the Beatmap Nominators website](https://bn.mappersguild.com/), called *BN Finder*.
+  - Anyone can submit maps in the BN Finder queue that's present on [the home page](https://bn.mappersguild.com/home) of the website.
+  - Matches are made based on the preferences BNs/NATs can set.
+  - BNs/NATs can review potential matches below the submission form.
+  - After a match is accepted or denied, the [mappersguild bot](https://osu.ppy.sh/users/23648635) will send a message to the requester informing them of the consensus.
+
+### February
+
+- **2022-02-03:** Following [this discussion](https://osu.ppy.sh/community/forums/topics/1448027) about Beatmap Nominator activity, a bottom line has been added where the warning step would be skipped for activity checks if someone falls below it.
+  - Falling below the 90 day requirements with more than 60 days worth of nominations will result in an activity warning.
+  - Falling below 60 days worth of nominations in a 90 day period will result in a removal when activity is checked.
+- **2022-02-03:** osu!staff took Community Contributors nominations from the public for the first time via a form.
+- **2022-02-23:** `ModScore` activity over the past 3 months is no longer required to apply to the Beatmap Nominators in the osu!taiko, osu!catch, or osu!mania gamemodes.
+
+### March
+
+- **2022-03-01:** The osu!mania Nomination Assessment Team started adding more Asian members in order to improve the Asian represenation and accessibility to the Beatmap Nominators.
+  - In total, 3 new NAT members were added across the following months.
+- **2022-03-01:** Based on notable community interest and [a lengthy discussion](https://osu.ppy.sh/community/forums/topics/1325267), the osu!taiko Muzukashii break guideline has been reworked to allow for more freedom by adding a substitute to 3/2 breaks.
+  - This change has greatly increased the accessibility of mapping Muzukashii difficulties to more mappers.
+- **2022-03-23:** 15 Beatmap Nominators across all gamemodes were added to the [2021 Elite Nominators](https://osu.ppy.sh/home/news/2022-03-22-community-contributors-elite-nominators).
+- **2022-03-31:** Beatmap Nominator applications for the osu! gamemode have closed for the first time since the adoption of the new application system from the BN/QAT restructure.
+  - This was done in order to catch up with the constantly-growing backlog.
```

Comment:
```suggestion
- **2022-03-31:** Beatmap Nominator applications for the osu! game mode have closed for the first time, since the adoption of the new application system from the BN/QAT restructure.
  - This was done in order to catch up with the constantly-growing backlog.
```